### PR TITLE
optimze mastercontainer Dockerfile

### PR DESCRIPTION
**Pull request by Zoey2936**

### CODE REVIEW

File: Containers/mastercontainer/Dockerfile [hunk 1]
```diff
@@ -109,31 +99,23 @@ RUN sed -i \
...
-    chmod a+r /Caddyfile; \
-    chmod +x /healthcheck.sh
+COPY --chmod=664 Caddyfile /Caddyfile
+COPY --chmod=775 start.sh /usr/bin/start.sh
+COPY --chmod=775 backup-time-file-watcher.sh /backup-time-file-watcher.sh
+COPY --chmod=775 session-deduplicator.sh /session-deduplicator.sh
+COPY --chmod=775 cron.sh /cron.sh
+COPY --chmod=775 daily-backup.sh /daily-backup.sh
+COPY --chmod=664 supervisord.conf /supervisord.conf
+COPY --chmod=775 healthcheck.sh /healthcheck.sh
```
Comment (szaimen) [inline Containers/mastercontainer/Dockerfile:118]: can we put all of them into / and simply use one step for all of them by using an asterisk?

Comment (szaimen) [reply Containers/mastercontainer/Dockerfile:118]: Caddyfile and the supervisord probably need a step on their own

File: Containers/mastercontainer/Dockerfile [hunk 2]
```diff
@@ -7,22 +7,24 @@ FROM caddy:2.6.4-alpine as caddy
...
-    usermod -u 333 -g 333 xfs; \
-    groupmod -g 33 www-data; \
-    usermod -u 33 -g 33 www-data
-
 EXPOSE 80
 EXPOSE 8080
 EXPOSE 8443
 
-RUN mkdir -p /var/www/docker-aio;
+COPY mastercontainer.conf /etc/apache2/sites-available/mastercontainer.conf
```
Comment (szaimen) [inline Containers/mastercontainer/Dockerfile:14]: why is it up here?

File: Containers/mastercontainer/Dockerfile (hunk 1 above)
Comment (Zoey2936) [reply Containers/mastercontainer/Dockerfile]: done

File: Containers/mastercontainer/Dockerfile (hunk 2 above)
Comment (Zoey2936) [reply Containers/mastercontainer/Dockerfile:14]: because it was in the middle of RUN steps, I will put it down

Comment (Zoey2936) [reply Containers/mastercontainer/Dockerfile]: done

**Review by szaimen — APPROVED** [review]
LGTM

**Pull request merged**